controllers/watcher_controller.go

```diff
@@ -198,7 +211,7 @@ func (r *WatcherReconciler) updateWatcherStatusUsingSSA(ctx context.Context, wat
 }
 
 func (r *WatcherReconciler) initializeConditions(ctx context.Context, watcher *v1beta1.Watcher) error {
-	requiredCond := findWatcherCondition(watcher.Status.Conditions, v1beta1.WatcherConditionTypeReady)
+	requiredCond, _ := findWatcherCondition(watcher.Status.Conditions, v1beta1.WatcherConditionTypeVirtualService)
 	if requiredCond != nil {
```

Comment:
it's an initialization, you basically just need to create a new default condition, in later steps, update the condition accordingly.